packages/core/src/components/Alert/Alert.scss

```diff
@@ -22,20 +22,25 @@ Markup:
 Style guide: components.alert
 */
 
-$alert-icon-size: $spacer-5;
+$h-padding: $spacer-2;
+$bar-size: $spacer-1;
+$alert-icon-size: $spacer-7;
 
 .ds-c-alert {
   background-color: $color-primary-alt-lightest;
   background-image: url('#{$image-path}/info.png');
   background-image: url('#{$image-path}/info.svg');
-  background-position: $spacer-1 ($spacer-2 * 0.95);
+  background-position: $h-padding $h-padding;
   background-repeat: no-repeat;
   background-size: $alert-icon-size;
-  border: 1px solid $color-primary-alt-light;
   box-sizing: border-box;
   color: $color-base;
   min-height: $alert-icon-size + $spacer-2;
-  padding: $spacer-2;
+  padding-bottom: $h-padding * 0.7;
```

Comment:
Could we make the top and bottom padding consistent (`padding: $h-padding`)? Visually, this feels a little off.